packages/odf/components/kms-config/kms-config.tsx

```diff
@@ -90,22 +114,26 @@ export const KMSConfigure: React.FC<KMSConfigureProps> = ({
         label={t('Key management service provider')}
         className={`${className}__form-body`}
       >
-        <FormSelect
-          value={kmsProvider}
-          onChange={(_event, value) => setKMSProvider(value as ProviderNames)}
+        <Select
+          onSelect={(_event, value) => setKMSProvider(value as ProviderNames)}
+          toggle={toggle}
+          isOpen={isOpen}
           id="kms-provider"
-          name="kms-provider-name"
           aria-label={t('kms-provider-name')}
           isDisabled={isLengthUnity(allowedKMSProviders)}
         >
           {allowedKMSProviders.map((provider) => (
-            <FormSelectOption
-              value={provider.value}
-              label={provider.name}
-              key={provider.value}
-            />
+            <SelectOption value={provider.value} key={provider.value}>
+              {provider.value === ProviderNames.AZURE ? (
+                <>
+                  {provider.name} <TechPreviewBadge />
+                </>
+              ) : (
+                provider.name
+              )}
+            </SelectOption>
```

Comment:
what's the issue with this ?
```suggestion
            <FormSelectOption
              value={provider.value}
              label={<>{provider.name} <TechPreviewBadge /></>}
              key={provider.value}
            />
```

Reply:
FormSelectOptionProps.label is expecting a string